modules/core/misc/objc/common/CVObjcUtil.h

```diff
@@ -6,11 +6,13 @@
 
 #pragma once
 
+#ifdef HASH_UTILS
```

Comment:
I wonder, if those DOUBLE_TO_BITS, FLOAT_TO_BITS is necessary to have in the form of macros in header files? I guess, those conversions are not that performance-critical? Can we just declare those functions in the header file and implement, together with unions (btw, there are already such unions, Cv32suf and Cv64suf) somewhere in .cpp or .mm file? In this case, we don't have to put HASH_UTILS define in many headers

Reply:
@vpisarev thanks for pointing out about Cv32suf and Cv64suf - I have modified the PR to use them and removed the redundant V64 and V32 unions. I also moved the DOUBLE_TO_BITS and FLOAT_TO_BITS macros out to just the files that need them.